config/themePluginConfig.ts

```diff
@@ -0,0 +1,115 @@
+export default {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of default exports is forbidden](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=3667353)